errbot/backends/test.py

```diff
@@ -11,19 +11,6 @@
 
 log = logging.getLogger(__name__)
 
-# otherwise generate a temporary one.
-__import__('errbot.config-template')
-config = sys.modules['errbot.config-template']
-sys.modules['config'] = config
-
-tempdir = mkdtemp()
-config.BOT_DATA_DIR = tempdir
-config.BOT_LOG_FILE = tempdir + sep + 'log.txt'
-config.BOT_EXTRA_PLUGIN_DIR = []
-config.BOT_LOG_LEVEL = logging.DEBUG
-config.CHATROOM_PRESENCE = ('testroom',)  # we are testing with simple identfiers
-config.BOT_IDENTITY = {'username': 'err'}  # we are testing with simple identfiers
-
 # Errbot machinery must not be imported before this point
 # because of the import hackery above.
```

Comment:
It looks like this comment is no longer true now, so it can go away and the import statements below can go up with the other import statements (so then we can also remove the `noqa` statements behind them)